blog/_posts/2020-04-14-introducing-online-devops-dojo.md

```diff
@@ -3,67 +3,67 @@ title: "🐱‍👤 Introducing Online DevOps Dojo"
+Our challenge was to make the experience immersive. And to immerse youself in something, there is nothing better than a good story. So, we created one to support the learnings.
 
-The modules tell the story of a fictious company with its employees as they go through their DevOps journey.
+The modules tell the story of a fictious company, "Pet Clinic," and its employees as they go through their DevOps journey.
 
 ![Once Upon a Time](../../images/onceuponatime.jpg)
 
 Let us introduce you to some of the team members:
 
-<img src="../../images/brenda.png" width="400"/> | **B**renda from the **B**usiness who is keen to take the Pet Clinic in to new growth areas sees DevOps being key to that. She is extremely proud of her two golden retrievers Bonnie and Clyde.
-![Chun](../../images/chun.png) | **C**hun is a **c**oach, change facilitator and technologist. Chun is an avid outdoors person and adrenaline junky, lover of fun, hater of being on-call. She encourages teams to automate as much as possible of the build, test, package and deploy process to ensure that the systems were created with a LessOps model in mind.
-![Dan](../../images/dan.png) | **D**an, has worked as a senior developer for a number of years, most recently for First Order Incorporated a role he left due to the excessive travel requirements. Dan's workmates still fondly recall the fun and games which ensued when Shelob, his pet tarantula, got loose on *"Bring Your Pets to Work"* day.
+<img src="../../images/brenda.png" width="400"/> | **B**renda from the **B**usiness, who is keen to take the Pet Clinic into new growth areas, sees DevOps being key to that. She is extremely proud of her two golden retrievers Bonnie and Clyde.
+![Chun](../../images/chun.png) | **C**hun is a **c**oach, change facilitator and technologist. Chun is an avid outdoors person and adrenaline junky, lover of fun, hater of being on-call. She encourages teams to automate the build, test, package and deploy process as much as possible of to ensure that the systems are created with a LessOps model in mind.
+![Dan](../../images/dan.png) | **D**an has worked as a senior developer for a number of years, most recently for First Order Incorporated, a role he left due to the excessive travel requirements. Dan's workmates still fondly recall the fun and games that ensued when Shelob, his pet tarantula, got loose on *"Bring Your Pets to Work"* day.
 ![Adam](../../images/adam.png) | **A**dam is a System **A**dministrator transitioning to the role of Site Reliability Engineer (SRE). Adam is working closely with Selma to ensure the security of the company's software and applications. Adam suspects that Hal may have an inside man. Adam has a particular set of skills so he is working on a plan to look for the hacker, to find the hacker, to ... the hacker.

```

Comment:
```suggestion
![Adam](../../images/adam.png) | **A**dam is a System **A**dministrator transitioning to the role of Site Reliability Engineer (SRE). Adam is working closely with the Security Officer to ensure the security of the company's software and applications. Adam has a particular set of skills so he is working on a plan to look for the hacker, to find the hacker, to ... the hacker.

```

Reply:
Thanks a lot Lisa. Selma and Hal are described in the home page: https://dxc-technology.github.io/about-devops-dojo/
Instead of making another long list of personas in the blog post, I have removed the references to Selma and Hal names.